samples/embedded/embedded.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import os
```

Comment:
This file should (technically) also have a license header comment

Reply:
Thanks, done.